src/test/groovy/org/prebid/server/functional/tests/AuctionSpec.groovy

```diff
@@ -153,6 +156,73 @@ class AuctionSpec extends BaseSpec {
         assert bidderRequest?.user?.buyeruid == buyeruid
     }
 
+    def "PBS call to alias should populate bidder request buyeruid from family user.buyeruids when resolved name is present"() {
+        given: "Pbs config with alias"
+        def cookieName = PBSUtils.randomString
+        def prebidServerService = pbsServiceFactory.getService(PBS_CONFIG + GENERIC_CONFIG
+                + ["host-cookie.family"                          : GENERIC.value,
+                   "host-cookie.cookie-name"                     : cookieName,
+                   "adapters.generic.usersync.cookie-family-name": GENERIC.value,
+                   "adapters.generic.aliases.alias.enabled"      : "true",
+                   "adapters.generic.aliases.alias.endpoint"     : "$networkServiceContainer.rootUri/auction".toString()])
+
+        and: "Alias bid request"
+        def buyeruid = PBSUtils.randomString
+        def bidRequest = BidRequest.defaultBidRequest.tap {
+            imp[0].ext.prebid.bidder.alias = new Generic()
+            imp[0].ext.prebid.bidder.generic = null
+            ext.prebid.aliases = [ (ALIAS.value): bidderName]
```

Comment:
You don't need point out aliases in both place, can remove:
```
"adapters.generic.aliases.alias.enabled"      : "true",
"adapters.generic.aliases.alias.endpoint"     : "$networkServiceContainer.rootUri/auction".toString()
```

Reply:
agree, updated